### PR TITLE
MCKIN-11571 Translation change: Japanese

### PR DESCRIPTION
**Pull request by moeez96**
[MCKIN-11571](https://edx-wiki.atlassian.net/browse/MCKIN-11571)

Change ja translation for the `Submit` button as mentioned in the ticket.
Translations were not pushed to tranisfex.

### CODE REVIEW

**Comment by moeez96** [comment]
@xitij2000 Please review.

**Comment by msaqib52** [comment]
@moeez96 have approved the PR, don't forget to bump the version before the merge.

**Comment by moeez96** [comment]
@msaqib52 Thank you. I dont have access to merge or create a release version.

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 # Constants #########################################################
 
-VERSION = '3.3.9'
+VERSION = '3.4.0'
```
Comment (xitij2000) [inline setup.py]: Please increment the patch version since this is a minor change. So 3.3.10

**Comment by moeez96** [comment]
@xitij2000 updated patch version. Please review again.

**Pull request merged**